code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -449,7 +449,8 @@
 	else
 		for(var/obj/item/integrated_circuit/input/S in assembly_components)
 			S.attackby_react(I,user,user.a_intent)
-		return ..()
+		if(user.a_intent != INTENT_HELP)
+			return ..()
```

Comment:
Shouldn't this return TRUE otherwise ?

Reply:
my bad, I was referencing this line https://github.com/AverageJoe82/tgstation/blob/5927131c2ddce5677aa329a0a2cd4feb30b686f6/code/modules/integrated_electronics/core/assemblies.dm#L405 which I'll fix also

Reply:
After I made it true the circuit analyzer stopped working and it was apparently because it needs to return null or false for the afterattack() on items to run so I reverted the commit.